src/app2.js

```diff
@@ -16,19 +17,26 @@ class App extends React.Component {
       <Router>
         <div>
           <nav className="nav">
-            <h1>Bookstore CMS</h1>
+            <h1 className="nav-header">Bookstore CMS</h1>
```

Comment:
- [ ] Let's make sure that we're using the correct colors for our components. The title and the background of the app seem a bit off. This is your title:

![image](https://user-images.githubusercontent.com/79114233/146648952-1511aa19-c709-41a3-bd1e-2c9633440f16.png)

Reference:
![image](https://user-images.githubusercontent.com/79114233/146648960-c11dd56c-5eb3-43fc-80d9-39851a76c03e.png)

- [ ] Let's also correct the color from the bg, yours:
![image](https://user-images.githubusercontent.com/79114233/146648991-dbad5073-b671-4b15-b4e7-235158bfcf78.png)

Reference:
![image](https://user-images.githubusercontent.com/79114233/146649002-eacc702c-94dc-44b9-986c-caefbb4dda9b.png)

Reply:
Thank you for pointing them out. These have been resolved